cmd/segment-reaper/detect.go

```diff
@@ -25,9 +34,126 @@ var (
 		DatabaseURL string `help:"the database connection string to use" default:"postgres://"`
 		From        string `help:"begin of date range for detecting zombie segments" default:""`
 		To          string `help:"end of date range for detecting zombie segments" default:""`
+		File        string `help:"location of file with report" default:"detect_result.out"`
```

Comment:
Let's use .csv for extension. The file structure is of a CSV file, right?

Reply:
yeah, good point